normalize-git-url.js

```diff
@@ -23,8 +23,10 @@ module.exports = function normalize (u) {
   if (altered) {
     if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
       returnedUrl = u.replace(/^git\+(.*:[^:]+):(.*)/, '$1/$2')
+    } else if (u.match(/^git\+file/)) {
+      returnedUrl = u.replace(/^git\+/, '')
     } else {
-      returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '')
+      returnedUrl = u.replace(/^(?:git\+)?ssh:[\/\\]{2}/, '')
```

Comment:
oh. That. May have just been me thrashing around when first trying to fix this. I realized afterwards that slashes aren't just normalized quite that aggressively just because you're on windows :joy: -- I'll just take this bit off tbh.